src/transport/mqtt/config/tls.rs

```diff
@@ -45,7 +41,8 @@ pub(crate) fn is_env_ignore_ssl() -> bool {
 }
 
 pub(crate) struct ClientAuth {
-    certs: Vec<CertificateDer<'static>>,
+    pem: String,
+    der: CertificateDer<'static>,
```

Comment:
```suggestion
    der: Vec<CertificateDer<'static>>,
```

I was wrong in my previous suggestion, this should be an array, with certificates for multiple origins

Reply:
Did this ever happen and do we expect it to happen, have you asked the back-end team ? We can't base changes off intuition (i probably shouldn't have changed it the first time around) but I want to know if certificate for multiple origins are allowed first.